ChattoAdditions/Source/Chat Items/TextMessages/TextMessagePresenter.swift

```diff
@@ -50,8 +51,10 @@ open class TextMessagePresenter<ViewModelBuilderT, InteractionHandlerT>
                 sizingCell: sizingCell,
                 cellStyle: baseCellStyle
             )
+            self.menuPresenter = TextMessageMenuItemPresenter { [weak self] in self?.messageViewModel.text ?? "" }
     }
 
+    private var menuPresenter: ChatItemMenuPresenterProtocol!
```

Comment:
Please, don't use forced unwrapped optional. Could you use a delegate pattern or closure variable for late binding or smth similar?

Reply:
It should always be there. I cannot use let there because it requires access to `self`. I could make it optional and provide default values at the moment of usage if you want